### DISCUSSION TITLE
Renkli bilyeler sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#177] [Soru Cevap]
Kırmızı, mavi ve yeşil üç bilyeden;
1. A bilyesi kırmızı renklidir
2. B bilyesi kırmızı renkli değildir
3. C bilyesi mavi renkli değildir.

3 önermeden biri doğrudur. Buna göre renklerini bulunuz?

**Comment by Ersan0102** [comment]
**Hocam öncelikle özür dilerim, python'da programlama yapmayı biraz unutmuşum o yüzden size algoritmasını atıyorum.**
Eğer 1. yönerge doğru ise 2. yönergenin de mantıken doğru olması gerekir. Bu yüzden bu olamaz.
Eğer 1. yönerge yanlışsa 2. yönerge doğru olur. Bu durumda C bilyesi kırmızı olmak durumunda kalır. Bu sefer de 2 ve 3. yönergeler doğru olduğu için bu da olamaz.
Eğer eğer 1. yönerge ve 2. yönerge yanlışsa 3. yönerge doğru olur. Böylece B bilyesi kırmızı renkli,  A bilyesi mavi renkli(çünkü 3. yönerge doğru), C bilyesi de yeşil renkli olur.


**SONUÇ:**
A = Mavi
B = Kırmızı
C = Yeşil olur.
